extensions/roam-research/src/index.tsx

```diff
@@ -17,13 +18,25 @@ const GraphSelect = () => {
 };
 
 export default function Command() {
-  const [graphCache, saveGraphCache] = useGraphConfigCache();
+  const { graphsConfig, removeGraphConfig } = useGraphsConfig();
+  // TODO: PROBLEM is that this does not update when we add a new graph??
+
   const { push } = useNavigation();
   return (
     <List>
-      {graphList(graphCache, (graph) => {
-        push(<GraphDetail graph={graph} />);
-      })}
+      {graphList(
+        graphsConfig,
+        {
+          onAction: (graphConfig: GraphConfig) => {
+            push(<GraphDetail graphConfig={graphConfig} />);
+          },
+          title: "Available commands",
+        },
+        (graphName) => {
```

Comment:
`graphList` now takes an optional `removeGraphFunction` param and only shows action to "Remove graph" when this is passed